src/assets/i18n/de.json

```diff
@@ -11,8 +11,11 @@
   "REGISTER": "Registrieren",
   "RESETPASSWORD": "Passwort vergessen",
   "CANCEL": "Abbrechen",
+  "FORGOTPASSWORD": "Neues Passwort anfordern",
```

Comment:
Deinen neuen Einträge gehören alle zu "login", sollten entsprechend dort in dem nested Element untergebracht werden,